cmd/security-agent/subcommands/compliance/command.go

```diff
@@ -58,7 +58,7 @@ func complianceEventCommand(globalParams *command.GlobalParams) *cobra.Command {
 				fx.Supply(eventArgs),
 				fx.Supply(core.BundleParams{
 					ConfigParams: config.NewSecurityAgentParams(globalParams.ConfigFilePaths),
-					LogParams:    log.LogForOneShot(command.LoggerName, "info", true),
+					LogParams:    log.ForOneShot(command.LoggerName, "info", true),
```

Comment:
Unfortunately we (security-agent) wouldn't have control over the naming of an exported func from another library: https://github.com/DataDog/datadog-agent/blob/main/comp/core/log/params.go#L61

I do like how revive provides pointers to how to write more nicely formatted code, but I think we may need to slowly up the strictness of it.

Reply:
It's a bit annoying to rename it everywhere, but @DataDog/agent-shared-components owns it and they accepted the PR so I guess it's fine ?

Reply:
I did change it as well :) I'm trying to fix the last small issues that remain on the PR and will dispatch the PR differently to all teams